files/jsbarcode/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "JsBarcode",
+  "repo": "lindell/JsBarcode",
+  "files": {
+    "include": ["bin/browser/*.min.*"]
```

Comment:
Is this directory not required in production? https://github.com/lindell/JsBarcode/tree/v3.3.2/bin/browser/barcodes